lib/cogctl/action_util.ex

```diff
@@ -47,12 +73,20 @@ defmodule Cogctl.ActionUtil do
       {:ok, endpoint_with_token} ->
         fun.(endpoint_with_token)
       {:error, error} ->
+        error = case error do
+                  error when is_list(error) ->
+                    Enum.join(error, "\n")
+                  error ->
+                    inspect(error)
+                end
+
```

Comment:
The cond case doesn't read clearer to me in this case, but I'll take a look to see if there's anything else I might change there to make it clearer. At the very least, I probably should use different variable names instead of `error` everywhere.